### PR TITLE
Add Python-3.9.5-GCCcore-10.3.0.eb to NESSI/2022.11

### PR DESCRIPTION
**Pull request by TopRichard**
Trying to add Python-3.9.5-GCCcore-10.3.0-bare.eb to NESSI/2022.11

### CODE REVIEW

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -424,7 +424,7 @@ echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALL
 #$EB Perl-5.32.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2640
 #use enhanced CMake easyblock to patch CMake's UnixPaths.cmake script if --sysroot is set
 #from https://github.com/easybuilders/easybuild-easyblocks/pull/2248
-$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
+#$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:427]: Any reason to comment this out?

File: eessi-2022.11.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 easyconfigs:
   - EasyBuild-4.7.0.eb
-  - CMake-3.20.1-GCCcore-10.3.0.eb
```
Comment (trz42) [inline eessi-2022.11.yml:3]: Why removing it from the easystack file?

File: EESSI-pilot-install-software.sh [hunk 3]
```diff
@@ -449,6 +449,7 @@ $EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
 #####################
 ### add packages here
 #####################
+$EB Python-3.9.5-GCCcore-10.3.0-bare.eb --robot
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:452]: Wondering a bit about the specific easyconfig `-bare` suffix being used. Are there others (without the suffix)? If so why was this chosen?

File: eessi-2022.11.yml [hunk 4]
```diff
@@ -1,3 +1,3 @@
...
+  - Python-3.9.5-GCCcore-10.3.0-bare.eb
```
Comment (trz42) [inline eessi-2022.11.yml:3]: See comment about suffix `-bare`.

**Review by trz42 — CHANGES_REQUESTED** [review]
Left a few comments.

File: EESSI-pilot-install-software.sh (hunk 3 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh:452]: Couldn't find in (easybuild-easyconfigs) other than that for version 10.3.0, any suggestions?

File: eessi-2022.11.yml (hunk 4 above)
Comment (TopRichard) [reply eessi-2022.11.yml:4]: Followed the guide lines.

File: EESSI-pilot-install-software.sh (hunk 3 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh]: Found the Python version WITHOUT the bare- after assistance, now it is just Python-3.9.5-GCCcore-10.3.0

File: eessi-2022.11.yml (hunk 2 above)
Comment (TopRichard) [reply eessi-2022.11.yml:3]: Added it back and followed the guidelines

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh]: Followed the new guidelines and added it back

**Comment by trz42** [comment]
Build for `skylake_avx512` failed. Checking on the build cluster (Saga) it seemed as this might have been just a temporary/local issue (builds for `cascadelake` and `generic` succeeded). We retrigger the build hoping it works out this time.

**Comment by trz42** [comment]
All built and looking good. Starting ingestion procedure.

**Review by trz42 — APPROVED** [review]

**Comment by trz42** [comment]
All ingested and available online. Ready to merge.

**Pull request merged**